prog/kubernetes/kubernetes_node_nexus.rb

```diff
@@ -26,11 +26,49 @@ def before_run
   end
 
   label def wait
+    when_retire_set? do
+      hop_retire
+    end
+    when_remove_node_from_cluster_set? do
+      hop_remove_node_from_cluster
+    end
     nap 6 * 60 * 60
   end
 
+  label def drain
+    unit_name = "drain_node_#{kubernetes_node.name}"
```

Comment:
why do we need a special unit_name that contains the node name here, instead of a static name?